Makefile

```diff
@@ -1,7 +1,9 @@
 CFLAGS= -lcriterion -lm -Wall -Wextra -Wpedantic -Wcast-align -Winit-self -Wmissing-include-dirs -Wredundant-decls -Wshadow -Wstrict-overflow=5 -Wundef -Wwrite-strings -Wpointer-arith -Wmissing-declarations -Wuninitialized -Wold-style-definition -Wstrict-prototypes -Wmissing-prototypes -Wswitch-default -Wbad-function-cast -Wnested-externs -Wconversion -Wunreachable-code
 build: Test.c
+	gcc main.c dciCommon.c -O2 -o decoder.out $(CFLAGS)
```

Comment:
Why build main.c in label "build: Test.c" ?

Reply:
This is test for compilation of main.c ?

Reply:
Yes, it is. We dont have full developed main.c, and we just test compilation and execution one.